src/test/java/org/cactoos/io/OutputStreamToTest.java

```diff
@@ -54,17 +53,15 @@ public void writesLargeContentToFile() throws IOException {
         MatcherAssert.assertThat(
             "Can't copy Input to Output and return Input",
             new TextOf(
-                new TeeInput(
-                    new ResourceOf("org/cactoos/large-text.txt"),
-                    new OutputTo(new OutputStreamTo(temp))
+                new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.